tutorials/horizontally-scale-mysql-database-backend-with-google-cloud-sql-and-proxysql/index.md

```diff
@@ -0,0 +1,325 @@
+	LOAD MYSQL SERVERS TO RUNTIME;
+	SAVE MYSQL SERVERS TO DISK;
+	```
+
+	`OFFLINE_SOFT` status flag instructs ProxySQL to stop sending new queries to the read replica. Existing connections still persist. Monitor the status of the read replica in Cloud Console (Go to the [Cloud SQL Instances page](https://console.cloud.google.com/sql/instances), click the read replica you would like to remove, and the performance chart is available in the Instance Details page). You should see that your read replica is no longer used by your application. Note that setting the read replica to offline in ProxySQL does not stop it from running in Cloud SQL; you are still being charged for the replica. See steps below to remove it from Cloud SQL.
+
+2. If you do not plan to use the read replica any more in the future, delete it from `mysql_servers` table:
+
+	```
+	DELETE FROM mysql_servers WHERE hostname = '[IP_ADDRESS]';
+	LOAD MYSQL SERVERS TO RUNTIME;
+	SAVE MYSQL SERVERS TO DISK;
+	```
+
+	All the traffic will be terminated immediately, so you should proceed with deletion only after you are certain that the read replica has completed all the queries sent its way. Use the performance chart (`MySQL queries`, `active connections`, etc.) to monitor the status.
+
+	Then remove it from Cloud SQL:
+
+		a. Go to the [Cloud SQL Instances page](https://console.cloud.google.com/sql/instances) and click the read replica you would like to remove.
+		b. Click **Delete**. Follow the instructions on screen to delete the instance.
+
+3. Update the rules and weights accordingly.
+
+Unlike the master instance, read replicas cannot be stopped (and then restarted later) in Cloud SQL. If you plan to add a read replica after deletion, create a new one and update ProxySQL settings. Master instance cannot be stopped or deleted until all the read replicas are deleted.
+
+## Cleaning Up
+
+After finishing this tutorial, you can clean up the resources you created on Google Cloud Platform so that you will not be billed for them in the future. To clean up, you can delete the whole project or delete the master instance in Cloud SQL. 
+
+### Deleting the Project
+
+Visit the [Manage resources menu](https://console.cloud.google.com/cloud-resource-manager). Select the project you used for this tutorial and click **Delete**. Note that once the project is deleted, the project ID cannot be reused.
+
+If you have Cloud SDK installed in the system, you can also [use the `gcloud` command-line to delete a project](https://cloud.google.com/sdk/gcloud/reference/projects/delete).
+
+### Deleting the Master Instance
+
+Go to the [Cloud SQL Instances page](https://console.cloud.google.com/sql/instances). If there are still read replicas, remove one by one first by clicking the kebab icon (three vertical dots) and choose **Delete** in the drop down menu. After all the read replicas are removed, delete the master instance in the same manner.
```

Comment:
Suggested edit: "If there are still read replicas, remove them one by one. For each one, click the kebab icon (three vertical dots) and choose **Delete** in the drop down menu."